shell/platform/common/accessibility_bridge.cc

```diff
@@ -43,7 +43,30 @@ void AccessibilityBridge::AddFlutterSemanticsCustomActionUpdate(
 }
 
 void AccessibilityBridge::CommitUpdates() {
+  // AXTree cannot move a node in a single update.
+  // This must be split across two updates:
+  //
+  // * Update 1: remove nodes from their old parents.
+  // * Update 2: re-add nodes (including their children) to their new parents.
+  //
+  // First, start by removing nodes if necessary.
+  std::optional<ui::AXTreeUpdate> remove_reparented =
+      CreateRemoveReparentedNodesUpdate();
+  if (remove_reparented.has_value()) {
+    tree_.Unserialize(remove_reparented.value());
+
+    std::string error = tree_.error();
+    if (!error.empty()) {
+      FML_LOG(ERROR) << "Failed to update ui::AXTree, error: " << error;
+      assert(false);
```

Comment:
Does `assert(false)` do anything?

Reply:
Yup, it crashes if this is a debug build but no-ops on a release build. In other words, it forces us to investigate this bug while minimizing the impact to our customers :)